man/ocamlrun.m

```diff
@@ -200,6 +200,10 @@ Startup messages (loading the bytecode executable file, resolving
 .BR 0x200
 Computation of compaction-triggering condition.
 
+.BR 0x400
+Output full GC statistics at program exit (as per Gc.quick_stat,
+with the same format as Gc.print_stat)
```

Comment:
I think this description a bit confusing: we have the "full stats", but they are "per `quick_stat`", and with the format of `print_stat`? Maybe we could just say, "Output GC statistics at program exit". If you need to specify the format as you expect external users to rely on it, adding `(with the same format as Gc.print_stat)` would be fine. Saying that they are "full" but only the quick_stat are actually given does not bring much (and we may want to extend the statistics given in the future).